src/Display/Modal/Modal.tsx

```diff
@@ -73,7 +74,7 @@ export const Modal: FC<Props> = ({
 
     if (isVisible) {
       // On modal open
-      scrollableTarget.focus();
```

Comment:
Have tried `scrollableTarget.tabindex = 0` but it didn't work. So I just revert the change. https://github.com/glints-dev/glints-aries/pull/608/files#diff-88e4bf70b17929c5c287f91a7c665ed684dcf98c47a7e2b72c3ea003a82b9e81L65